storage_service/locations/models/fixity_log.py

```diff
@@ -0,0 +1,28 @@
+# stdlib, alphabetical
+
+# Core Django, alphabetical
+from django.db import models
+
+# Third party dependencies, alphabetical
+from django_extensions.db.fields import UUIDField
+
+# This project, alphabetical
+
+# This module, alphabetical
+
+
+class FixityLog(models.Model):
+    """ Stores fixity check success/failure and error details """
+
+    package = models.ForeignKey('Package', to_field='uuid')
+    success = models.BooleanField(default=False)
+    error_details = models.TextField()
+    time_reported = models.DateTimeField(auto_now=True)
```

Comment:
Nitpick, but the `DateTimeField` is called `time` here and ` date` in Package. For consistency, call them both the same thing. Maybe datetime?